Sources/SwiftCurrent_UIKit/StoryboardLoadable.swift

```diff
@@ -70,6 +70,18 @@ extension StoryboardLoadable {
     }
 }
 
+@available(iOS 13.0, *)
+extension PassthroughFlowRepresentable where Self: StoryboardLoadable {
+    /// :nodoc: **WARNING: This will throw a fatal error.** Just a default implementation of the required `FlowRepresentable` initializer meant to satisfy the protocol requirements.
+    public init(with args: WorkflowInput) { // swiftlint:disable:this unavailable_function
+        // swiftlint:disable:next line_length
+        fatalError("The StoryboardLoadable protocol provided a default implementation if this initializer so that consumers didn't have to worry about it in their UIViewController. If you encounter this error and need this initializer, simply add it to \(String(describing: Self.self))")
```

Comment:
```suggestion
        fatalError("The StoryboardLoadable protocol provided a default implementation of this initializer so that consumers didn't have to worry about it in their UIViewController. If you encounter this error and need this initializer, simply add it to \(String(describing: Self.self))")
```

Reply:
No need, this whole init is gone now